dapr/clients/http/client.py

```diff
@@ -16,6 +16,9 @@
 import aiohttp
 
 from typing import Callable, Mapping, Dict, Optional, Union, Tuple, TYPE_CHECKING
+
+from dapr.clients.grpc._helpers import parse_endpoint
```

Comment:
Let's move this somewhere else. We should not import anything from `grpc` subpaths in the `http` client. That doesn't make any sense :)